cache/contenthash/tarsum.go

```diff
@@ -37,10 +37,10 @@ func v0TarHeaderSelect(h *tar.Header) (orderedHeaders [][2]string) {
 
 func v1TarHeaderSelect(h *tar.Header) (orderedHeaders [][2]string) {
 	pax := h.PAXRecords
-	if len(h.Xattrs) > 0 { // field deprecated in stdlib
+	if len(h.Xattrs) > 0 { //nolint:staticcheck // field deprecated in stdlib
```

Comment:
Looks like https://github.com/moby/buildkit/pull/5256/files#r1716936764 was some kind of bug in 1.60